backend/interview_portal/src/main/java/by/interview/portal/dto/DisciplineDTO.java

```diff
@@ -19,7 +17,5 @@
 
     private Long parentId;
 
-    private String parentName;
-
-    private Set<UserBaseInfoDTO> disciplineHeadsList;
+    private boolean hasSubItems;
```

Comment:
I recommend using wrapper-objects(boolean - Boolean)

Reply:
I disagree here. Why do you recommend this? Any reasons?